### ISSUE TITLE
Did not download Enigma, and/or didn't start it.

### ISSUE THREAD

**Issue by jeffreymarkbaldridge** [asiekierka/MinecraftPOMF#675]
I ran the yarn as per instructions, the command prompt specified 5 actionable tasks: 5 executed, but Enigma is nowhere to be seen.  Do I need to install Enigma before hand?  From your Readme, it indicated that it would be downloaded.  Upon running the yarn again I got 4 executed 1 up-to-date.  It did download and create the merged jar.  Any sort of nudge / hand holding in the right direction would be most appreciated.  Thank you.

**Comment by liach** [comment]
Is the final `yarn` task up to date?

**Comment by jeffreymarkbaldridge** [comment]
Unfortunately, my wife needed to take the laptop to work, so I cannot
confirm anything at the moment.  If I recall correctly, when I ran it, the
program started downloading the mc jars, once it completed that it merged
them, then gave me a warning that it was using depreciated functions and
couldn't run gradle 5.0, and then said it was done.  There was a single
line that said 5 executables, and 4 were finish and 1 was up to date.  Do
you know where it downloads enigma?  I checked in most of the files in my
yarn install, but didn't see anything that looked like it would be enigma.
